src/_data/sidenav.yml

```diff
@@ -1,38 +1,65 @@
-- title: Get started
+- header: Get started
+- title: Installation
   children:
     - title: Install Flutter
       permalink: /get-started/install
     - title: Test drive
       permalink: /get-started/test-drive
+
+
+- title: Learn Flutter
+  children:
+    - title: Introduction
+      permalink: /get-started/learn-flutter
+    - title: Dart language overview
+      permalink: https://dart.dev/overview
```

Comment:
I like keeping the link to Dart, but do you think it should be after the course content? It depends who you are targeting with the new material, but if it's developers who already know to program in a similar language, consider putting this entry after.

Also, maybe we should link to [Bootstrap into Dart](https://docs.flutter.dev/resources/bootstrap-into-dart) instead? Then we can update that page and curate it more to the Flutter first week experience as well.

Reply:
I had similar thoughts, I'll make those changes.